test/extended/builds/pipeline_origin_bld.go

```diff
@@ -274,33 +274,41 @@ var _ = g.Describe("[Feature:Builds][Feature:Jenkins][Slow] openshift pipeline b
 			})
 
 			g.By("should verify services successfully", func() {
-				// instantiate the bc
-				g.By("create the jenkins pipeline strategy build config that leverages openshift client plugin")
-				err := oc.Run("new-app").Args("-f", verifyServiceClientPluginPipelinePath).Execute()
-				o.Expect(err).NotTo(o.HaveOccurred())
 
-				err = exutil.WaitForDeploymentConfig(oc.KubeClient(), oc.AppsClient().AppsV1(), oc.Namespace(), "nodejs-example", 1, false, oc)
+				newTestAppArgs := []string{"--template", "redis-ephemeral", "-p", "MEMORY_LIMIT=128Mi"}
```

Comment:
minor change, but, can you just rename the newTestAppArgs into newAppRedisEphemeralArgs?

Reply:
Can you also add the --name=redis parameter, just to ensure that the name is explicitly set here? You can use a local variable to no repeat the hardcoded name in the subsequent calls.